py/make_requests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
this is the top level program to do a bulk import. it basically goes to s3 and starts downloading data in the background while its making requests to the reporter in the foreground